webapp/src/components/modals/actionModal.tsx

```diff
@@ -0,0 +1,173 @@
+import React, {useEffect, useState} from 'react';
+
+import Button from 'react-bootstrap/Button';
+import Modal from 'react-bootstrap/Modal';
+import Table from 'react-bootstrap/Table';
+import Form from 'react-bootstrap/Form';
+
+import './styles.css';
+
+import {OverlayTrigger, Tooltip} from 'react-bootstrap';
+
+type Props = {
+    visibility: boolean;
+    setVisibility: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
I don't see any use of these in the component.

Reply:
Check if these can be removed

Reply:
it is used to hide the modal when close button in clicked, i don't think we can remove it